llvm/docs/ReleaseNotes.md

```diff
@@ -214,6 +214,9 @@ Changes to LLDB
   Windows 11 on the Microsoft SQ2 and Snapdragon Elite X platforms.
 * LLDB now steps through C++ thunks. This fixes an issue where previously, it
   wouldn't step into multiple inheritance virtual functions.
+* A statusline was added to command-line LLDB to show progress events and
```

Comment:
"This is on by default and can be configured using the `whatever the enable setting is` and `whatever the format setting is` settings."